code/game/machinery/navbeacon.dm

```diff
@@ -107,6 +100,13 @@
 	else
 		return ..()
 
+/obj/machinery/navbeacon/screwdriver_act(mob/living/user, obj/item/I)
+
+	. = TRUE
```

Comment:
I think it's probably better to explicitly return TRUE at the end of these to reduce confusion. Attack chain stuff is confusing enough as it is.